rust-mode.el

```diff
@@ -2121,6 +2115,12 @@ visit the new file."
                (delete-pair))
               (t (rust-insert-dbg)))))))
 
```

Comment:
I think that this can be removed.

Reply:
Maybe, I don't know.
I don't think my commits touch this.
This should be addressed separately.